suite-native/device-manager/src/components/DeviceList.tsx

```diff
@@ -142,7 +152,11 @@ export const DeviceList = ({ isVisible, onSelectDevice }: DeviceListProps) => {
 
     return (
         <Animated.View style={[listAnimatedStyle, applyStyle(listStaticStyle)]}>
-            <VStack>
+            <View
+                style={applyStyle(contentWrapperStyle, {
```

Comment:
Could no this be styled few lines lower in the conditional render? IMO it would be more readable since this style is applied under the same condition as the conditional render. I would prefer to isolate the conditional changes to single place.

Reply:
good thinking. changed in f72721d0be